1_dice_roler/gotta_go_fast/andrerfcsantos/go/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.out
+*.exe
+*.prof
+main
+bonus
```

Comment:
Just add a newline at the EOF

Reply:
There is a global gitignore which has golang already, so add just the binaries in this one.

Reply:
I added this just because a global gitgnore didn't exist at the time. I implemented and pushed both requested changes to all my current branches wit PR's.

Even though there is a global gitignore with golang now, i suggested some improvements in #6. It's safe to merge this branch without that particular pull request, but it will be important for future go code.